neutron/assetlist.json

```diff
@@ -528,6 +528,32 @@
       "images": [
         {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/bad.png"
+        },
+        {
+      "description": "BULLDOG",
+      "denom_units": [
+        {
+          "denom": "neutron1fe4fd33d0976gqr833xhdxmw5hv8c5s4nv7pvdrdflpr8u4uadjq76tmdq",
+          "exponent": 0,
+          "aliases": [
+            "ubulldog"
+          ]
+        },
+        {
+          "denom": "bulldog",
+          "exponent": 6
+        }
+      ],
+      "base": "neutron1fe4fd33d0976gqr833xhdxmw5hv8c5s4nv7pvdrdflpr8u4uadjq76tmdq",
+      "name": "BULLDOG",
+      "display": "bulldog",
+      "symbol": "BDG",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/bulldog.svg"
```

Comment:
you have an svg in the png property...
Also, can you find an image that uses transparent background instead of white background?